docs/pages/blog/tenerife-retreat.md

```diff
@@ -0,0 +1,162 @@
+date: 2022-07-25T00:00:00.000Z
+authors: ['samuelsycamore']
+card: true
+tags: ['Company']
+---
+
+One of the toughest challenges to overcome as a fully remote team is fostering a supportive and inclusive company culture.
+How do you build trust, comradery, and—dare I say it?—_friendships_, when there are so few opportunities for spontaneous social interactions in our daily work routines?
+
+At MUI, we've chosen to prioritize in-person company retreats as a way to grow together and cultivate the culture that informs how we operate.
+
+![Photo of the MUI team in front of the pool at our accommodations in Tenerife](/static/blog/tenerife-retreat/group-photo.jpeg)
+
+In late June 2022, sixteen of us gathered at a remote private residence on the island of Tenerife.
+This was MUI's second company retreat, the first being in Portugal in 2021 when the team was just a fraction of the size it is today.
+
+We had a ton of fun getting to know each other better over oceanic activities, poolside conversations, and incredible catered meals—not to mention a steady supply of Spanish wine. 🥂
+Oh, and I guess we got some work done along the way, too? 😅
+
+## Destination
+
+![Satellite image of Tenerife](/static/blog/tenerife-retreat/tenerife.jpeg)
+
+Tenerife is a volcanic island off the coast of western Africa that's part of the Canary Islands (Spanish: _Canarias_).
+It is the most populous island in the archipelago, and one of the most visited tourist destinations in the world.
+Its central defining feature is Mount Teide, the third largest volcano in the world when measured from its base, which rises 7,500 m (24,600 ft) from the bottom of the ocean floor.
+
+With an average temperature ranging from about 18-25 °C (68-77 °F) year-round, it's hard to pick a bad time to visit!
+
+## Accommodations
+
+![Photo of the pool in the evening at Symbiosis Living](/static/blog/tenerife-retreat/pool-evening.jpeg)
+
+[Symbiosis Living](https://www.symbiosisliving.com/) is a sustainable retreat venue nestled deep in the hills on the western side of the island.
+The venue is a venerable oasis of lush green flora amid the stark and barren volcanic rock that surrounds it, hosting breezy guest houses adorned with rustic, regionally sourced furniture.
+
+![Photo of the outdoor kitchen at Symbiosis Living](/static/blog/tenerife-retreat/outdoor-kitchen.jpeg)
+
+The owners were exceptionally accommodating and supportive for the entirety of our stay.
+Almost all of our meals were cooked fresh by the hosts in their fabulous outdoor kitchen.
+I don't think I've ever eaten that many incredible home-cooked meals in a single week. 😋
+And somehow each one was even better than the last!
+
+![MUI X colleagues José and Andrew bond over a bottle of wine at dinner one night during the retreat.](/static/blog/tenerife-retreat/jose-andrew-wine.jpg)
+
+_MUI X colleagues José and Andrew bond over a bottle of wine at dinner one night during the retreat._
```

Comment:
```suggestion
<p class="blog-description">MUI X colleagues José and Andrew bond over a bottle of wine at dinner one night during the retreat.</p>
```

You can also use this approach for adding descriptions! It gets a nice style 😊

Reply:
Cool, thanks! I'm thinking I ought to add these to all of the photos.